src/app/components/MediaLoader/configs/index.ts

```diff
@@ -0,0 +1,15 @@
+import { PageTypes } from '#app/models/types/global';
+import livePage from './livePage';
+import article from './article';
+
+export default (pageType: PageTypes) => {
+  switch (pageType) {
+    case 'live':
+      return livePage;
+    case 'article':
+    case 'mediaArticle':
+      return article;
+    default:
+      return null;
+  }
```

Comment:
We can see how this approach plays out as we extend the new media player to other pagetypes.

I'm curious how the likes of live streams will come through on the Live pages, as currently only the `article` config builder checks for `webcastVersions`.

This separation is ok for now I feel, but if page types start mixed OD and live streams, we may need to reconsider this approach and select the correct config builder based on media type (e.g. `aresMedia` or `clipMedia`), rather than page type.

Reply:
This task focuses on Live pages anyway, so adding in the `article` config builder was just more for testing the theory on this approach.